packages/shylu/shylu_dd/bddc/src/shylu_DofManager.h

```diff
@@ -92,28 +93,29 @@ template <class LO, class GO>
   {
     Tpetra::global_size_t IGO =
       Teuchos::OrdinalTraits<Tpetra::global_size_t>::invalid();
-    RCP<const Map> nodeMap =
-      rcp( new Map(IGO,
-                   Teuchos::ArrayView<const GO>(nodeGlobalIDs, numNodes),
-                   0, Comm));
+    RCP<const Map> nodeMap = 
+      rcp( new Map(IGO, 
+		   Teuchos::ArrayView<const GO>(nodeGlobalIDs, numNodes), 
+		   0, Comm));
     RCP<const Map> nodeMap1to1;
     if (nodeGlobalIDs1to1 == nullptr) {
       nodeMap1to1 = Tpetra::createOneToOne<LO,GO,Node>(nodeMap);
     }
     else {
-      nodeMap1to1 =
-        rcp( new Map(IGO, Teuchos::ArrayView<const GO>(*nodeGlobalIDs1to1),
-                     0, Comm) );
+      nodeMap1to1 = 
+	rcp( new Map(IGO, Teuchos::ArrayView<const GO>(*nodeGlobalIDs1to1), 
+		     0, Comm) );
     }
-    // The rows and columns of the graph nodeGraphSubdomain are node global IDs
+    // The rows and columns of the graph nodeGraphSubdomain are node global IDs 
     // and local degrees of freedom (dofs), respectively, prior to assembly.
     RCP<CrsGraph> nodeGraphSubdomain;
-    determineNodalDofGraph(numNodes, nodeBegin, localDofs, Comm, nodeMap,
-                           nodeMap1to1, nodeGraphSubdomain);
-    // The graph nodeGraph1to1 is the counterpart of graph nodeGraphSubdomain,
+    determineNodalDofGraph(numNodes, nodeBegin, localDofs, Comm, nodeMap, 
+			   nodeMap1to1, nodeGraphSubdomain);
+    // The graph nodeGraph1to1 is the counterpart of graph nodeGraphSubdomain, 
     // but with uniquely owned rows. Note: it is assumed that the local dofs
     // for each node are identical across all processors
-    RCP<CrsGraph> nodeGraph1to1 = rcp( new CrsGraph(nodeMap1to1, 0) );
+    size_t maxNumCols = nodeGraphSubdomain->getGlobalMaxNumRowEntries();
```

Comment:
It would be more efficient to set this bound per row, but this will work.  The only issue is if you have a small number of dense rows.

Reply:
I ended up removing the + line above and simply passed 0 to the CrsGraph constructor in the following line (looks like this may be what I did originally).